code/client/clrcore/External/PedHash.cs

```diff
@@ -6,411 +6,104 @@ namespace CitizenFX.Core
-		Jesus01 = 3459037009u,
-		Jewelass01 = 4040474158u,
-		JewelSec01 = 2899099062u,
-		JewelThief = 3872144604u,
-		Justin = 2109968527u,
-		Mani = 3367706194u,
-		Markfost = 479578891u,
-		Marston01 = 943915367u,
-		MethFemale01 = 3778572496u,
-		MethMale01 = 1293671805u,
-		MilitaryBum = 1191548746u,
-		Miranda = 1095737979u,
-		Mistress = 1573528872u,
-		Misty01 = 3509125021u,
-		MovieStar = 894928436u,
-		MPros01 = 1822283721u,
-		Niko01 = 4007317449u,
-		Paparazzi = 1346941736u,
-		Party01 = 921110016u,
-		PartyTarget = 2180468199u,
-		PestContDriver = 994527967u,
-		PestContGunman = 193469166u,
-		Pogo01 = 3696858125u,
-		Poppymich = 602513566u,
-		Princess = 3538133636u,
-		Prisoner01 = 2073775040u,
-		PrologueHostage01 = 3306347811u,
-		PrologueMournFemale01 = 2718472679u,
-		PrologueMournMale01 = 3465937675u,
-		RivalPaparazzi = 1624626906u,
-		ShopKeep01 = 416176080u,
-		SpyActor = 2886641112u,
-		SpyActress = 1535236204u,
-		StripperLite = 695248020u,
-		Taphillbilly = 2585681490u,
-		Tramp01 = 1787764635u,
-		VagosFun01 = 3299219389u,
-		WillyFist = 2423691919u,
-		WeedFemale01 = 1596374223u,
-		WeedMale01 = 2648833641u,
-		Zombie01 = 2890614022u,
 		Acult01AMM = 1413662315u,
 		Acult01AMO = 1430544400u,
 		Acult01AMY = 3043264555u,
 		Acult02AMO = 1268862154u,
 		Acult02AMY = 2162532142u,
 		AfriAmer01AMM = 3513928062u,
+		Agatha = 1855569864u,
+		AgathaCutscene = 756308504u,
+		Agent = 610988552u,
+		Agent02 = 1183124263u,
+		Agent14 = 4227433577u,
+		Agent14Cutscene = 1841036427u,
+		AgentCutscene = 3614493108u,
 		Airhostess01SFY = 1567728751u,
 		AirworkerSMY = 1644266841u,
-		Ammucity01SMY = 2651349821u,
+		AlDiNapoli = 4042020578u,
+		Alan = 1925887591u,
+		AmandaTownley = 1830688247u,
+		AmandaTownleyCutscene = 2515474659u,
 		AmmuCountrySMM = 233415434u,
+		Ammucity01SMY = 2651349821u,
+		Andreas = 1206185632u,
+		AndreasCutscene = 3881194279u,
+		AnitaCutscene = 117698822u,
+		AntonCutscene = 2781317046u,
+		Antonb = 3479321132u,
 		ArmBoss01GMM = 4058522530u,
 		ArmGoon01GMM = 4255728232u,
 		ArmGoon02GMY = 3310258058u,
 		ArmLieut01GMM = 3882958867u,
+		Armoured01 = 3455013896u,
 		Armoured01SMM = 2512875213u,
 		Armoured02SMM = 1669696074u,
 		Armymech01SMY = 1657546978u,
+		Ary = 3473698871u,
+		Ary02 = 2275358319u,
+		Ary02Cutscene = 3927407837u,
+		AryCutscene = 3059505486u,
+		Ashley = 2129936603u,
+		AshleyCutscene = 650367097u,
 		Autopsy01SMY = 2988916046u,
+		Autoshop01SFM = 12394276u,
 		Autoshop01SMM = 68070371u,
 		Autoshop02SMM = 4033578141u,
+		Autoshop03 = 109850898u,
+		Avery = 3088269167u,
+		AveryCutscene = 1427949869u,
+		AviSchwartzman = 939183526u,
+		AviSchwartzmanCutscene = 2560490906u,
+		Avischwartzman02 = 3735478533u,
+		Avischwartzman02Cutscene = 2315189472u,
+		Avon = 4242698434u,
+		AvonCutscene = 406009421u,
+		Avongoon = 2618542997u,
 		Azteca01GMY = 1752208920u,
+		Babyd = 3658575486u,
 		BallaEast01GMY = 4096714883u,
 		BallaOrig01GMY = 588969535u,
-		Ballas01GFY = 361513884u,
```

Comment:
The alphanumeric ordering started so nicely, what happened here? Think I also saw some other occurrences down below.